package.json

```diff
@@ -2,14 +2,42 @@
   "name": "classnames",
   "version": "2.4.0",
   "description": "A simple utility for conditionally joining classNames together",
-  "main": "index.js",
   "author": "Jed Watson",
   "license": "MIT",
   "repository": {
     "type": "git",
     "url": "https://github.com/JedWatson/classnames.git"
   },
+  "type": "commonjs",
```

Comment:
Added this explicit `type` field as well, which will help eliminate guesswork in future Node.js versions.